backend/routes/chat_routes.py

```diff
@@ -30,72 +30,63 @@ def fetch_user_stats(commons, user, date):
     return userItem
 
 # get all chats
-@chat_router.get("/chat", dependencies=[Depends(AuthBearer())])
+@chat_router.get("/chat", dependencies=[Depends(AuthBearer())], tags=["Chat"])
 async def get_chats(commons: CommonsDep, current_user: User = Depends(get_current_user)):
+    """
+    Retrieve all chats for the current user.
+
+    - `current_user`: The current authenticated user.
+    - Returns a list of all chats for the user.
+
+    This endpoint retrieves all the chats associated with the current authenticated user. It returns a list of chat objects
+    containing the chat ID and chat name for each chat.
+    """
     date = time.strftime("%Y%m%d")
     user_id = fetch_user_id_from_credentials(commons, date, {"email": current_user.email})
     chats = get_user_chats(commons, user_id)
     return {"chats": chats}
 
 # get one chat
-@chat_router.get("/chat/{chat_id}", dependencies=[Depends(AuthBearer())])
+@chat_router.get("/chat/{chat_id}", dependencies=[Depends(AuthBearer())], tags=["Chat"])
 async def get_chats(commons: CommonsDep, chat_id: UUID):
+    """
+    Retrieve details of a specific chat by chat ID.
+
+    - `chat_id`: The ID of the chat to retrieve details for.
+    - Returns the chat ID and its history.
+
+    This endpoint retrieves the details of a specific chat identified by the provided chat ID. It returns the chat ID and its
+    history, which includes the chat messages exchanged in the chat.
+    """
     chats = get_chat_details(commons, chat_id)
     if len(chats) > 0:
         return {"chatId": chat_id, "history": chats[0]['history']}
     else:
         return {"error": "Chat not found"}
 
+
+
 # delete one chat
-@chat_router.delete("/chat/{chat_id}", dependencies=[Depends(AuthBearer())])
+@chat_router.delete("/chat/{chat_id}", dependencies=[Depends(AuthBearer())], tags=["Chat"])
 async def delete_chat(commons: CommonsDep, chat_id: UUID):
+    """
+    Delete a specific chat by chat ID.
+    """
     delete_chat_from_db(commons, chat_id)
     return {"message": f"{chat_id}  has been deleted."}
 
-# helper method for update and create chat
```

Comment:
Hey, Thanks for your contribution. 

I am confused why you seem to have removed a lot of the logic for these routes?

Reply:
seems to have gone wrong ill fix it thanks for the review!